### ISSUE TITLE
Theme doesn't work correctly

### ISSUE THREAD

**Issue by Toadfield** [Swordfish90/cool-retro-term#637]
It looks like this in settings:
![image](https://user-images.githubusercontent.com/68649672/105494803-a1e96780-5cbb-11eb-8288-0b33f430b08c.png)

**Comment by Lateralus138** [comment]
Yep, mine looks like this as well. I don't use this term often, but I like it, just not much time to look into the issue myself it or I'd fork it. Maybe later this year.

**Comment by MrAureliusR** [comment]
Having the same issue, glad I found an existing issue to bump instead of opening a new one.

![image](https://user-images.githubusercontent.com/41264289/163668478-becd02f8-ce77-4e3a-b2d8-6440e0a4566e.png)

I'm using KDE with the default Dark Breeze theme on Arch Linux. This is the only window in any app that I've seen act like this. I'd be happy to help debug the issue in any way, but with the huge number of open issues... well, let's hope this gets some attention eventually :)

This isn't my main terminal emulator, or else I'd probably roll my sleeves up and try and track down the bug myself, but I just don't have the time right now.

**Comment by MrAureliusR** [comment]
Okay, I've done a bit of digging. I cloned the repo, and built the latest commit, which has the issue. Then I checked out 1.1.1 which, while it does have a white border in the settings menu, is perfectly readable and normal. Then I used bisect to narrow down the commit, and it was commit dae1a56691841cbc37d597c5da59bfbc91e1e1a9 which caused the issue. So moving from QtQuick.Controls 1.x to 2.x *or* updating the QMLTermWidget submodule is what caused the issue.

Hope this helps a bit, I might try playing around to see exactly which change caused the issue.

**Comment by Darthpenguin** [comment]
I thought I was having this issue becasue I'm using the Cinnamon desktop instead of KDE plasma. But I guess the problem exists in KDE also.

**Comment by wd5gnr** [comment]
Try quick workaround: launch like this:

QT_QUICK_CONTROLS_STYLE=Material cool-retro-term

Ugly, but you can read it. This bug is in a few different places.

**Comment by denilsonsa** [comment]
FYI, these styles serve as workarounds:

    QT_QUICK_CONTROLS_STYLE=Imagine cool-retro-term
    QT_QUICK_CONTROLS_STYLE=Material cool-retro-term
    QT_QUICK_CONTROLS_STYLE=Universal cool-retro-term
